recipes/flask_cors/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "Flask-Cors" %}
+{% set pname = "flask_cors" %}
+{% set version = "3.0.8" %}
+
+package:
+  name: {{ pname|lower }}
```

Comment:
In general, you don't need to have the `pname` variable. For the package name, you can actually use `name: {{ name|lower|replace("-", "_") }}`. This will lower the letters from `name` (Flask-Cors), and replace `-` with `_`.